### PR TITLE
Fix deprecations

### PR DESCRIPTION
**Pull request by Revent**
change unused modules and functions in future releases of SDK.

### CODE REVIEW

File: src/org/intellij/erlang/quickfixes/ErlangExportFunctionFix.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import com.intellij.ui.components.JBList;
 import com.intellij.ui.popup.HintUpdateSupply;
 import com.intellij.util.containers.ContainerUtil;
+import kotlin.reflect.jvm.internal.impl.utils.SmartList;
```
Comment (ignatov) [inline src/org/intellij/erlang/quickfixes/ErlangExportFunctionFix.java]: That's wrong SmartList, be careful. We should use com.intellij.util.SmartList.

File: tests/org/intellij/erlang/refactoring/ErlangSafeDeleteTest.java [hunk 2]
```diff
@@ -18,9 +18,9 @@
 
 import com.intellij.psi.PsiElement;
 import com.intellij.refactoring.safeDelete.SafeDeleteHandler;
-import com.intellij.testFramework.fixtures.LightCodeInsightFixtureTestCase;
+import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase;
 
-public class ErlangSafeDeleteTest extends LightCodeInsightFixtureTestCase {
+public class ErlangSafeDeleteTest extends LightJavaCodeInsightFixtureTestCase {
```
Comment (ignatov) [inline tests/org/intellij/erlang/refactoring/ErlangSafeDeleteTest.java]: We shouldn't use LightJavaCodeInsightFixtureTestCase, better to find a language-agnostic test case.

File: tests/org/intellij/erlang/debugger/ErlangSourcePositionTest.java [hunk 3]
```diff
@@ -28,6 +29,7 @@
 public class ErlangSourcePositionTest extends ErlangLightPlatformCodeInsightFixtureTestCase {
   private static final String MODULE_NAME = "erlang-source-position";
   private ErlangFile myErlangFile;
+  private Module myModule;
```
Comment (ignatov) [inline tests/org/intellij/erlang/debugger/ErlangSourcePositionTest.java]: Why we should have a module here?

File: tests/org/intellij/erlang/completion/ErlangCompletionTestBase.java [hunk 4]
```diff
@@ -35,6 +36,8 @@
 import java.util.List;
 
 abstract public class ErlangCompletionTestBase extends ErlangLightPlatformCodeInsightFixtureTestCase {
+  private Module myModule;
```
Comment (ignatov) [inline tests/org/intellij/erlang/completion/ErlangCompletionTestBase.java]: The same question about the module.

File: src/org/intellij/erlang/quickfixes/ErlangExportFunctionFix.java (hunk 1 above)
Comment (Revent) [reply src/org/intellij/erlang/quickfixes/ErlangExportFunctionFix.java]: sorry fixed

File: tests/org/intellij/erlang/refactoring/ErlangSafeDeleteTest.java (hunk 2 above)
Comment (Revent) [reply tests/org/intellij/erlang/refactoring/ErlangSafeDeleteTest.java]: my wrong using BasePlatformTestCase

File: tests/org/intellij/erlang/debugger/ErlangSourcePositionTest.java (hunk 3 above)
Comment (Revent) [reply tests/org/intellij/erlang/debugger/ErlangSourcePositionTest.java]: don't know fix it.

File: tests/org/intellij/erlang/completion/ErlangCompletionTestBase.java (hunk 4 above)
Comment (Revent) [reply tests/org/intellij/erlang/completion/ErlangCompletionTestBase.java]: don't know fix it

File: src/org/intellij/erlang/inspection/ErlangUndefinedCallbackFunctionInspection.java [hunk 5]
```diff
@@ -25,6 +25,7 @@
 import com.intellij.psi.codeStyle.CodeStyleManager;
 import com.intellij.util.ObjectUtils;
 import com.intellij.util.containers.ContainerUtil;
+import kotlin.reflect.jvm.internal.impl.utils.SmartList;
```
Comment (kvakvs) [inline src/org/intellij/erlang/inspection/ErlangUndefinedCallbackFunctionInspection.java]: Nope, no Kotlin

File: src/org/intellij/erlang/psi/impl/ErlangFileImpl.java [hunk 6]
```diff
@@ -35,6 +35,7 @@
 import com.intellij.util.containers.ContainerUtil;
 import com.intellij.util.containers.MultiMap;
 import gnu.trove.THashMap;
+import kotlin.reflect.jvm.internal.impl.utils.SmartList;
```
Comment (kvakvs) [inline src/org/intellij/erlang/psi/impl/ErlangFileImpl.java]: No Kotlin

File: src/org/intellij/erlang/refactoring/introduce/ErlangExtractFunctionHandler.java [hunk 7]
```diff
@@ -42,6 +42,7 @@
 import com.intellij.refactoring.util.CommonRefactoringUtil;
 import com.intellij.util.ObjectUtils;
 import com.intellij.util.containers.ContainerUtil;
+import kotlin.reflect.jvm.internal.impl.utils.SmartList;
```
Comment (kvakvs) [inline src/org/intellij/erlang/refactoring/introduce/ErlangExtractFunctionHandler.java]: No Kotlin

**Pull request closed without merging**